zingo-testutils/src/lib.rs

```diff
@@ -681,6 +681,69 @@ pub mod scenarios {
         )
     }
 
+    pub async fn funded_orchard_with_3_txs_mobileclient(
```

Comment:
Why do we include the work `mobileclient` in this `ident`?  Is there anything about this code that's specific to a `mobileclient`?